pkg/agent/evictionmanager/plugin/memory/system_pressure_test.go

```diff
@@ -177,54 +161,6 @@ func TestSystemPressureEvictionPlugin_ThresholdMet(t *testing.T) {
 			wantIsUnderSystemPressure: false,
 			wantSystemAction:          actionNoop,
 		},
-		{
```

Comment:
why delete these test cases?

Reply:
Because detectSystemKswapdStealPressure() was disabled in the patch.